editor/src/clj/editor/pipeline/bob.clj

```diff
@@ -28,12 +29,21 @@
     [editor.prefs :as prefs]
     [editor.workspace :as workspace])
   (:import
-    [com.dynamo.bob ClassLoaderScanner IProgress IResourceScanner Project TaskResult]
+    [com.dynamo.bob Bob ClassLoaderScanner IProgress IResourceScanner Project TaskResult]
     [com.dynamo.bob.fs DefaultFileSystem]
     [com.dynamo.bob.util PathUtil]
+    [java.io File InputStream PrintStream PrintWriter PipedInputStream PipedOutputStream]
+    [java.net URI]
+    [java.nio.charset StandardCharsets]
     [org.apache.commons.io FilenameUtils]
-    [java.io File InputStream]
-    [java.net URI]))
+    [org.apache.commons.io.output WriterOutputStream]))
+
+(try
+  (doto (.getDeclaredField Bob "verbose")
+    (.setAccessible true)
+    (.setBoolean nil true))
+  (catch Exception e
+    (throw (Exception. "Couldn't make Bob verbose" e))))
```

Comment:
Can we not achieve this by adding `"verbose" "true"` in the `generic-bundle-bob-args` function? Or maybe calling `(.setOption "verbose" "true")` in the `bob-build!` function if we want to force it everywhere?

Reply:
Bob sets `verbose` to true only in the `Bob::mainInternal` static method that also calls `System.exit()` at the end, so I had to go through reflection.